app/javascript/components/pages/FateShow.js

```diff
@@ -1,24 +1,43 @@
-import React, { Component } from 'react'
-import Button from '../components/Button'
+import React, { Component } from "react";
+import Button from "../components/Button";
 
 class FateShow extends Component {
-
-    render() {
-        return (
-            <>
-                <div>
-                    <h1>Show Page</h1>
-                    {this.props.event && (
-                        <div>
-                            <p>{this.props.event.category}</p>
-                            <p>{this.props.event.description}</p>
-                            <img src={this.props.event.image_url} />
-                        </div>
-                    )}
-                </div>
-            </>
-
-        )
-    }
+  handleDelete = () => {
+    fetch(`/events/${this.props.event.id}`, {
+      headers: { "Content-Type": "application/json" },
+      method: "DELETE",
+    })
+      .then((response) => {
+        if (response.status > 400) {
+          return response.status;
```

Comment:
these two returns in this fetch are doing nothing... if you see on line 17 in the next `.then` you're not doing anything with this info. You should handle the case if the response is an error by setting some kind of state showing that something went wrong or just `console.error(`response status: ${response.status})` or something at least... since it's a DELETE, you really dont need any of the `.thens` if you don't care about the response...

Reply:
fixed now should console log an error